src/traces/surface/convert.js

```diff
@@ -249,10 +245,30 @@ proto.update = function(data) {
         dynamicColor: [[1, 1, 1, 1], [1, 1, 1, 1], [1, 1, 1, 1]],
         dynamicWidth: [1, 1, 1],
         dynamicTint: [1, 1, 1],
-        opacity: 1,
-        colorBounds: [data.zmin * scaleFactor[2], data.zmax * scaleFactor[2]]
+        opacity: 1
+        // TODO: Need to think about how to calculate this
```

Comment:
[`calcColorscale`](https://github.com/plotly/plotly.js/blob/master/src/components/colorscale/calc.js), which gets called before the convert step, will take care of this.